### PR TITLE
Fixed error to save scroll position

### LINKED ISSUE #1711: Chatroom doesn't save the previous scroll position
### Problem

When a user enters the chatroom, it should scroll to the last position where the user was viewing.
But it doesn't save the previous scroll position and only scrolls to the bottom.
It only happens in **e2e-protocol** branch, not **master** branch.

### Solution

WORK ON **e2e-protocol** branch or other branch which is created from it, and fix it.

### PR DESCRIPTION
**Pull request by Silver-IT**
### Summary of Changes

- Closes #1711

### CODE REVIEW

**Comment by Silver-IT** [comment]
@taoeffect, in order to work on the latest version of code regarding the chatroom, I have branched off on [this branch](https://github.com/okTurtles/group-income/tree/1697-remove-private-dms-group-dms-everywhere-and-use-directmessages-instead).
So I think this PR should be merged after the [PR #1709](https://github.com/okTurtles/group-income/pull/1709).

File: frontend/model/contracts/identity.js [hunk 1]
```diff
@@ -61,7 +61,9 @@ sbp('chelonia/defineContract', {
       process ({ data }, { state }) {
         const initialState = merge({
           settings: {},
-          attributes: {},
+          attributes: {
+            allowDMInvite: true
+          },
```
Comment (taoeffect) [inline frontend/model/contracts/identity.js]: Can you merge `e2e-protocol` back into this PR?

**Review by taoeffect — CHANGES_REQUESTED** [review]
Preliminary review

**Review by taoeffect — APPROVED** [review]
Seems to work! Great job @Silver-IT !

**Pull request merged**